skdecide/hub/space/gym/gym.py

```diff
@@ -420,7 +420,7 @@ def get_elements(self) -> Iterable[T]:
         return self._elements
 
     def sample(self) -> T:
-        return self._elements[super().sample()]
+        return self._to_elements[super().sample()]
```

Comment:
is it a bug you found during this PR implementation ?

Reply:
Correct, I found this bug when calling the `sample` method of `SetSpace` in the UP bridge test units.